### PR TITLE
Enable account dropdown in narrow sidebar menu

### PR DESCRIPTION
**Pull request by amyspark**
This pull request lets the user access the account dropdown even with the narrow menu.
I've also adjusted the CSS to use flex everywhere.

### CODE REVIEW

File: src/renderer/components/TimelineSpace/SideMenu.vue [hunk 1]
```diff
@@ -199,6 +202,15 @@ export default {
           color: #409eff;
         }
       }
+    }
+
+    .profile-wide {
+      color: #ffffff;
```
Comment (h3poteto) [inline src/renderer/components/TimelineSpace/SideMenu.vue]: __[stylelint]__ <sub>posted by Sider</sub>

Expected "#ffffff" to be "#fff" (color-hex-length)

File: src/renderer/components/TimelineSpace/SideMenu.vue [hunk 2]
```diff
@@ -207,18 +219,26 @@ export default {
...
+      padding-bottom: 2px;
 
       .avatar {
-        width: 36px;
-        height: 36px;
-        border-radius: 50%;
-        margin: 18px 12px;
+        display: inline;
+
+        img{
```
Comment (h3poteto) [inline src/renderer/components/TimelineSpace/SideMenu.vue]: __[stylelint]__ <sub>posted by Sider</sub>

Expected single space before "{" (block-opening-brace-space-before)

**Comment by h3poteto** [comment]
Could you show any sample image? I want to confirm the appearance.

**Comment by amyspark** [comment]
Sure! I took these in a Windows box. (They are the same whether one shows or not the account sidebar.)

These show the side menu. You'll see the wide one hasn't changed at all:
wide-menu
narrow-menu

These show the dropdown control.
wide-menu dropdown open
narrow-menu dropdown open

**Pull request merged**